src/main/java/com/minecolonies/coremod/blocks/BlockWaypoint.java

```diff
@@ -0,0 +1,93 @@
+package com.minecolonies.coremod.blocks;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+
+import java.util.Locale;
+
+import static com.minecolonies.api.util.constant.Suppression.DEPRECATION;
+
+/**
+ * This block is used as a substitution block for the Builder.
+ * Every solid block can be substituted by this block in schematics.
+ * This helps make schematics independent from location and ground.
+ */
+public class BlockWaypoint extends AbstractBlockMinecolonies<BlockWaypoint>
+{
+
+    /**
+     * The hardness this block has.
+     */
+    private static final float BLOCK_HARDNESS = 0.0F;
+
+    /**
+     * This blocks name.
+     */
+    private static final String BLOCK_NAME = "blockWayPoint";
+
+    /**
+     * The resistance this block has.
+     */
+    private static final float RESISTANCE = 1F;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public BlockWaypoint()
+    {
+        super(Material.WOOD);
+        initBlock();
+    }
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(Locale.ENGLISH), BLOCK_NAME));
+        setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        setHardness(BLOCK_HARDNESS);
+        setResistance(RESISTANCE);
+    }
+
+    /**
+     * Used to determine ambient occlusion and culling when rebuilding chunks
+     * for render.
+     * @deprecated
+     *
+     * @return true
+     */
+    //todo: remove once we no longer need to support this
+    @SuppressWarnings(DEPRECATION)
+    @Override
+    @Deprecated
+    public boolean isOpaqueCube(final IBlockState state)
+    {
+        return false;
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @SuppressWarnings(DEPRECATION)
+    @Override
+    @Deprecated
+    public boolean isFullBlock(final IBlockState state)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)